docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,119 @@
+# Move an instance from one region to another
+
+This guide will show you how to move or copy your instance volume to a different region in {{extra.brand}} Cloud.
```

Comment:
In #64 we dropped the `extra` prefix from macros, so you can just put `in {{brand}} Cloud.`